package-lock.json

```diff
@@ -1,12 +1,11 @@
 {
-  "name": "react-query-lite",
+  "name": "build-your-own-tanstack-query",
```

Comment:
Thanks for the suggestion!

I removed and reinstalled package-lock.json file on #22.